packages/renderer/src/components/EditorPage/component.tsx

```diff
@@ -204,8 +224,9 @@ export function EditorPage() {
               <Button
                 color="primary"
                 disabled={loadingPublish || isInstallingProject}
-                onClick={handleOpenModal('publish')}
+                onClick={handleOpenPublishModal}
                 startIcon={loadingPublish ? <Loader size={20} /> : <PublicIcon />}
+                // extra={<PublishOptions onClick={handleClickPublishOptions} />}
```

Comment:
this is commented out since the UX for the feature is done but we are waiting for the server side implementation